src/LayerEditor/ui/data/blocks_model.py

```diff
@@ -0,0 +1,29 @@
+from LayerEditor.ui.data.block_item import BlockItem
+from LayerEditor.ui.tools.id_map import IdMap
+from gm_base.geometry_files.format_last import InterpolatedNodeSet
+
+
+class BlocksModel:
+    def __init__(self, geo_model, le_model):
+        self.blocks = IdMap()
```

Comment:
This constructor is more like a factory function (static method), 
for this and possibly other simple wrappers around a IdMap 
it may be better to just use IdMap + a construction function at upper layer. 
This class adds nearly nothing to IdMap.

Reply:
It is not a wrapper of IdMap. It only uses IdMap as colection of object with ids. ."..Model" objects are managing underlying "...Item" objects. In this branch I created framework for upcoming changes. Therefore now this object has a lot more functionality inside.